kuma/payments/jinja2/payments/email/thank_you/email.html

```diff
@@ -142,12 +142,11 @@ <h3>{{_('Managing your payments')}}</h3>
                                                         <tbody>
                                                           <tr>
                                                             <td align="left" style="font-family:Geneva,Tahoma,Verdana,sans-serif;font-size:14px;line-height:22px;color:#555555;padding-top:16px">
-                                                              <h3>{{_('Cancelling your payments')}}</h3>
+                                                              <h3>{{_('Refunds and cancelling your payments section')}}</h3>
```

Comment:
Is "section" required here?

Reply:
sorry my mistake :( I also fix same issue in thank you page :)